addon/components/one-way-input.js

```diff
@@ -86,12 +86,16 @@ const OneWayInputComponent = Component.extend({
   },
 
   init() {
-    this._super(...arguments);
```

Comment:
No, I think you are correct: http://rwjblue.jsbin.com/vexodub/edit?html,js,output

This was listed as fixed in the CHANGELOG for 2.0.1 though, so I must have screwed something up :disappointed: